.github/workflows/mirror-images-regsync.yml

```diff
@@ -0,0 +1,64 @@
+name: Mirror Images Using regsync
+
+on:
+  workflow_dispatch: {}
+
+jobs:
+  mirror-images:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Check for secrets being set by workflow_dispatch
+        id: check_manual_secrets
+        run: |
+          if [ -n "${{ secrets.DEBUG_DOCKER_USERNAME }}" ]; then
+            echo "manual_secrets=true" >> $GITHUB_OUTPUT
+          elif [ -n "${{ secrets.DEBUG_DOCKER_PASSWORD }}" ]; then
+            echo "manual_secrets=true" >> $GITHUB_OUTPUT
+          elif [ -n "${{ secrets.DEBUG_APPCO_USERNAME }}" ]; then
+            echo "manual_secrets=true" >> $GITHUB_OUTPUT
+          elif [ -n "${{ secrets.DEBUG_APPCO_PASSWORD }}" ]; then
+            echo "manual_secrets=true" >> $GITHUB_OUTPUT
+          elif [ -n "${{ secrets.DEBUG_PRIME_USERNAME }}" ]; then
+            echo "manual_secrets=true" >> $GITHUB_OUTPUT
+          elif [ -n "${{ secrets.DEBUG_PRIME_PASSWORD }}" ]; then
+            echo "manual_secrets=true" >> $GITHUB_OUTPUT
+          else
+            echo "manual_secrets=false" >> $GITHUB_OUTPUT
+          fi
```

Comment:
is this something that needs to be committed, or is this just for debugging in PR CI?

Reply:
This is for developing the workflow. Specifically, its for setting the secrets when running the workflow in one's personal fork - there is no way to do this when fetching the secrets via the EIO action.

Do you not want it there? I suppose this can be added back if we need to develop further, but IMO it's nice to have for when things break in the future. Certainly not ideal, but then again, neither is github actions...

Reply:
I'm a little sketched out by having stuff that's not used by GHA, in the GHA config. Is there any reason you can't just set the non-DEBUG env vars when you're running the tests locally?